includes/class-activitypub.php

```diff
@@ -108,8 +109,22 @@ public static function render_json_template( $template ) {
 		if ( \is_author() ) {
 			$json_template = ACTIVITYPUB_PLUGIN_DIR . '/templates/author-json.php';
 		} elseif ( is_comment() ) {
+			$comment     = \get_comment( \get_query_var( 'c', null ) );
```

Comment:
@janboddez we now check that twice... here and in the template...

Reply:
@janboddez we also check this in the template, so we do the transformation twice! Maybe we can remove the check here or at least simplify it?!?